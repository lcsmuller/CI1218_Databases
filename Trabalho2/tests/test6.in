88 30 R A
89 31 R B
90 31 W B
91 32 R C
92 33 R C
93 33 W C
94 32 R C
95 30 C -
96 31 C -
97 32 C -
98 33 C -
