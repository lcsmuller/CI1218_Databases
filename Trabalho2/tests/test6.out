1 30,31,32,33 NS NV
